client/src/components/Carousel.jsx

```diff
@@ -11,20 +11,22 @@ class Carousel extends React.Component {
     this.state = {
       show: false,
       index: 0,
```

Comment:
Like you had said earlier...a few weeks from now you might forget what index is.  A more descriptive name would be  better.